README.md

```diff
@@ -854,6 +854,43 @@
         .call(tron.led);
     ```
 
+  - Leave a blank line after blocks and before the next statement
+
+    ```javascript
+    // bad
+    if (true) {
+      var a = 2;
+    }
+    var b = 3;
+
+    // good
+    if (true) {
+      var a = 2;
```

Comment:
Sure, It's a bad example. Haha, I can fix that, but the point here is to discuss the spacing between lines after the closing curly braces. How do you feel about that? :)